### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by Aditya-171**

### CODE REVIEW

File: docs/guide/evaluator.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ the [Pusher](pusher.md) that it is ok to push the model to production.
 
 *   Consumes:
     *   An eval split from
-        [ExampleGen](https://www.tensorflow.org/tfx/guide/examplegen)
+        [ExampleGen](https://www.tensorflow.org/tfx/guide/examplegen) or from [Transform](https://www.tensorflow.org/tfx/guide/transform)
```
Comment (lego0901) [inline docs/guide/evaluator.md:18]: I think this one could be more accurate and less wordy if it is
`[Examples](https://www.tensorflow.org/tfx/api_docs/python/tfx/v1/types/standard_artifacts/Examples)`.

File: tfx/components/evaluator/component.py [hunk 2]
```diff
@@ -65,7 +65,7 @@ def __init__(
 
     Args:
       examples: A BaseChannel of type `standard_artifacts.Examples`, usually
-        produced by an ExampleGen component. _required_
+        produced by an ExampleGen component or by Transform component. _required_
```
Comment (lego0901) [inline tfx/components/evaluator/component.py:68]: Saying "usually", I think describing only the ExampleGen would be fine enough.

**Comment by lego0901** [comment]
Hi @Aditya-171, thanks for your consideration toward TFX!

To be concise, there are many standard components that are able to produce Examples artifact. For example, ExampleGen family (CSVExampleGen, ImportExampleGen), Transform as you mentioned, and also BulkInferrer can generate it if `output_example_spec` argument is provided).

I thought mentioning all components might be too wordy, and evaluating Examples artifact generated by the Transform or BulkInferrer is not the general use cases. Therefore, I would like to use the original version comments of [tfx/components/evaluator/component.py](https://github.com/tensorflow/tfx/pull/6265/files/19f1fcfc45d6bcd01c30636dfdcd907ffe010e3c#diff-68f6a565e37114926c6f47108535e7b15d6aabd5c17fd6a7140f78862e636ecb).

However, [docs/guide/evaluator.md](https://github.com/tensorflow/tfx/pull/6265/files/19f1fcfc45d6bcd01c30636dfdcd907ffe010e3c#diff-50b978eb1e55cfabf65bc4514ce512dfdbf6664502c7dd72fa48bdbf368fc0d2) should be better to be changed. Could you specify the **Examples artifact** instead of its producer components?

I appreciate your consideration.

**Comment by lego0901** [comment]
+ Please rename the title of PR to describe what you did better "Patch 1" -> "..."

File: docs/guide/evaluator.md (hunk 1 above)
Comment (Aditya-171) [reply docs/guide/evaluator.md:18]: Agreed

**Comment by Aditya-171** [comment]
@lego0901 
Request to review.

**Comment by lego0901** [comment]
LGTM! Thanks

**Review by lego0901 — APPROVED** [review]
